Cubical/Foundations/Isomorphism.agda

```diff
@@ -105,3 +105,26 @@ compIso (iso fun inv rightInv leftInv) (iso fun₁ inv₁ rightInv₁ leftInv₁
   = cong fun₁ (rightInv (inv₁ b)) ∙ (rightInv₁ b)
 compIso (iso fun inv rightInv leftInv) (iso fun₁ inv₁ rightInv₁ leftInv₁) .Iso.leftInv a
   = cong inv (leftInv₁ (fun a) ) ∙ leftInv a
+
+idIso : ∀ {ℓ} {X : Set ℓ} → Iso X X
+Iso.fun (idIso {X = X}) = idfun X
+Iso.inv (idIso {X = X}) = idfun X
+Iso.rightInv (idIso {X = X}) x = refl {x = x}
+Iso.leftInv (idIso {X = X}) x = refl {x = x}
+
+-- Helpful notation
+_Iso⟨_⟩_ : ∀ {ℓ ℓ' ℓ''} {B : Set ℓ'} {C : Set ℓ''} (X : Set ℓ) → Iso X B → Iso B C → Iso X C
```

Comment:
I think shift-iso would look nicer with this notation. (unless you find a direct proof that is)

Reply:
Should I cange `_Iso⟨_⟩_` to `_≃⟨_⟩_` ?

Reply:
We already use that for equivalences, so I think this is good